deployments/data102/image/apt.txt

```diff
@@ -0,0 +1,94 @@
+# Some linux packages for basic terminal work, particularly
+# oriented at users new to Unix/cmd line environments.
+
+# Basic unix tools
+man
+man-db
+manpages-posix
+manpages-dev
+manpages-posix-dev
+
+# Download tools
+curl
+wget
+
+# Core text editors on a *nix box: vim and emacs
+vim
+emacs-nox
+emacs-goodies-el
+python-mode
```

Comment:
the build is failing on this for some reason, but it's almost certainly fine to remove and unblock the PR.

Reply:
Thanks